src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double GROWING_COEFFICIENT = 1.5;
+    private static final int DEFAULT_LENGTH = 10;
+    private int size;
+    private T[] elementData;
+
+    public ArrayList() {
+        elementData = (T[]) new Object[DEFAULT_LENGTH];
+    }
+
     @Override
     public void add(T value) {
-
+        elementData = growIfArrayFull();
```

Comment:
grow method should be void
```suggestion
        elementData = growIfArrayFull();
```